From 288d9e66cb34dfa19a6190b0fd4bfb0cdb08018b Mon Sep 17 00:00:00 2001
Message-Id: <288d9e66cb34dfa19a6190b0fd4bfb0cdb08018b.1413836944.git.chang-joon.lee@intel.com>
In-Reply-To: <5d8aa9994fff43965a6fab8c9a528b8b47a9d624.1413836944.git.chang-joon.lee@intel.com>
References: <5d8aa9994fff43965a6fab8c9a528b8b47a9d624.1413836944.git.chang-joon.lee@intel.com>
From: Arun Siluvery <arun.siluvery@linux.intel.com>
Date: Wed, 15 Oct 2014 10:21:23 +0100
Subject: [PATCH 66/71] REVERTME [VPG]: drm/i915: Add missing semicolon to
 avoid warnings

A semicolon is missing in i915_sync_create_fence() definition causing
compile errors when CONFIG_DRM_I915_SYNC flag is not enabled.
The original commit is https://android.intel.com/#/c/226486

NB: This is REVERTME because native sync is not accepted upstream.
There is work in progress to implement native sync on top of the linux
dma-sync framework instead.

Change-Id: Ia1a69b2c06ff3e28dd79d3080bcc6811d403d1e3
For: GMIN-2905
Cc: John Harrison <john.c.harrison@intel.com>
Signed-off-by: Arun Siluvery <arun.siluvery@linux.intel.com>
---
 drivers/gpu/drm/i915/intel_sync.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_sync.h b/drivers/gpu/drm/i915/intel_sync.h
index 50b6e0a..51fe6b2 100644
--- a/drivers/gpu/drm/i915/intel_sync.h
+++ b/drivers/gpu/drm/i915/intel_sync.h
@@ -99,7 +99,7 @@ void i915_sync_reset_timelines(struct drm_i915_private *dev_priv)
 static int i915_sync_create_fence(struct intel_engine_cs *ring, u32 seqno,
 				  int *fd_out)
 {
-	return 0
+	return 0;
 }
 
 static inline
-- 
1.7.9.5

